frontend/benefit/handler/src/components/applicationsArchive/ApplicationsArchive.tsx

```diff
@@ -105,6 +105,16 @@ const ApplicationsArchive: React.FC = () => {
     return cols.filter(Boolean);
   }, [t, getHeader, translationsBase]);
 
+  const sortBy = React.useMemo(
```

Comment:
isnt it supposed to be a backend ticket?

Reply:
hm there is a ticket for client side also.. https://helsinkisolutionoffice.atlassian.net/jira/software/c/projects/HL/boards/204?modal=detail&selectedIssue=HL-483